js/ui/lookup.d.ts

```diff
@@ -76,7 +76,7 @@ export type SelectionChangedEvent = EventInfo<dxLookup> & SelectionChangedInfo;
 export type TitleRenderedEvent = EventInfo<dxLookup> & TitleRenderedInfo;
 
 /** @public */
-export type ValueChangedEvent = NativeEventInfo<dxLookup> & ValueChangedInfo;
+export type ValueChangedEvent = NativeEventInfo<dxLookup, KeyboardEvent | MouseEvent | PointerEvent | UIEvent> & ValueChangedInfo;
```

Comment:
```suggestion
export type ValueChangedEvent = NativeEventInfo<dxLookup, KeyboardEvent | MouseEvent | PointerEvent | Event> & ValueChangedInfo;
```

https://github.com/DevExpress/DevExtreme/pull/19950/files#r741000230